### PR TITLE
HDDS-11228. Ozone Recon HeatMap refactoring of code.

### PR DESCRIPTION
**Pull request by devmadhuu**
## What changes were proposed in this pull request?
This PR change is only to refactor heatmap code to have better readability and maintainability.

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-11228

## How was this patch tested?

Patch tested manually.

### CODE REVIEW

**Comment by devmadhuu** [comment]
@ArafatKhan2198 kindly review. Just a refactor of existing code.

File: hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/AccessHeatMapEndpoint.java [hunk 1]
```diff
@@ -107,4 +107,10 @@ private static void checkIfHeatMapFeatureIsEnabled() {
       throw new WebApplicationException(Response.Status.NOT_FOUND);
     }
   }
+
+  @GET
+  @Path("/healthCheck")
```
Comment (ArafatKhan2198) [inline hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/AccessHeatMapEndpoint.java:112]: Can you please add a small java doc explaining the response and purpose of the endpoint.

**Review by ArafatKhan2198 — CHANGES_REQUESTED** [review]
Thanks for the patch @devmadhuu 
Overall, the refactoring looks good, just one comment to consider.

File: hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/AccessHeatMapEndpoint.java (hunk 1 above)
Comment (devmadhuu) [reply hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/AccessHeatMapEndpoint.java:117]: Thanks @ArafatKhan2198 for the review. Added javadoc now for API. Kindly re-review.

**Review by ArafatKhan2198 — APPROVED** [review]
Thanks for the changes @devmadhuu LGTM!

**Pull request merged**